test/blackbox-tests/test-cases/pkg/ocamlformat/patch-extra-files.t

```diff
@@ -0,0 +1,82 @@
+This is current bug, 'dune fmt' misses "dev-tools.locks" directory for the first run.
+
+  $ . ./helpers.sh
+  $ mkrepo
+
+Make a fake ocamlformat:
+  $ mkdir ocamlformat
+  $ cd ocamlformat
+  $ cat > dune-project <<EOF
+  > (lang dune 3.13)
+  > (package (name ocamlformat))
+  > EOF
+  $ cat > ocamlformat.ml <<EOF
+  > let () = print_string "Hello"
+  > EOF
+  $ cat > dune <<EOF
+  > (executable
+  >  (public_name ocamlformat))
+  > EOF
+  $ cd ..
+  $ tar -czf ocamlformat.tar.gz ocamlformat
```

Comment:
Nitpick, but if you don't compress the tarball then you avoid a dependency on `gzip`.

Reply:
Wouldn't that mean ocamlformat directory could not be removed ?

Reply:
I agree, opened #10952.

Reply:
Thanks.